chain/chunks/src/lib.rs

```diff
@@ -546,6 +546,15 @@ impl ShardsManager {
         let chunk_hash = one_part.chunk_hash.clone();
         let prev_block_hash = one_part.header.inner.prev_block_hash;
 
+        match self.runtime_adapter.get_epoch_id_from_prev_block(&prev_block_hash) {
```

Comment:
`if let` is better here

Reply:
My personal preference is if it's `Ok` I care about to do `if let`, if it's the error, to do `match`. `if let Err(err) = ...` looks weird to me.
Unless there's a strong objection.